timer/src/activities.ts

```diff
@@ -0,0 +1,27 @@
+// @@@SNIPSTART nodejs-timer-reminder-activity
+import { config } from 'dotenv';
+import mailgun from 'mailgun-js';
+
+config();
+
+const apiKey: string = process.env.MAILGUN_API as string;
```

Comment:
This cast is wrong, the type is string | undefined.
Instead of casting make a runtime assertion.

Reply:
Done using elvis operator, let me know if you prefer something else for backwards compat.